src/SDKs/_metadata/reservations_resource-manager.txt

```diff
@@ -0,0 +1,11 @@
+2019-10-28 03:47:35 UTC
```

Comment:
This file should not exist

Reply:
Should I delete it manually? All it should never be generated?

Reply:
This file was generated by `generate.cmd` right? That is discouraged and we recommend `generate.ps1`.
So please remove this file manually, and use `generate.ps1` for all future code generation. Please remove `generate.cmd` as well. Thanks!

Reply:
Please address this issue as well, thanks!

Reply:
deleted